src/Network/Network/help/Get-AzBastion.md

```diff
@@ -0,0 +1,719 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/get-azbastion
+schema: 2.0.0
+---
+
+# Get-AzBastion
+
+## SYNOPSIS
+Gets a Bastion resource or Bastion resources.
+
+## SYNTAX
+
+### ListBySubscriptionId (Default)
+```
+Get-AzBastion [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ListByResourceGroupName
+```
+Get-AzBastion [-ResourceGroupName <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ByResourceGroupNameByName
+```
+Get-AzBastion -ResourceGroupName <String> -Name <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### ByResourceId
+```
+Get-AzBastion -ResourceId <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-Bastion** cmdlet gets one or more bastions in a resource group or subscritption.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+Get-AzBastion
+```
```

Comment:
This may cause CI problems - generally you can only have one code section per example

Reply:
It should be fine to include the command and output in the same code section